src/components/EntityInfo.vue

```diff
@@ -139,8 +148,17 @@ export default class EntityInfo extends Mixins(EnumMixin) {
   readonly businessPhone!: string
   readonly businessPhoneExtension!: string
   readonly isRoleStaff!: boolean
+  readonly isLtd!: boolean
+  readonly isUlc!: boolean
   readonly nrNumber!: string
 
+  /** True if current company is LTD or ULC. */
+  private get isLtdOrUlc (): boolean {
+    // return (this.isLtd || this.isUlc)
+    // FOR NOW, FOR TESTING, LOOK FOR A SESSION STORAGE ITEM
+    return Boolean(sessionStorage.getItem('SHOW_COMPANY_INFO_BUTTON'))
```

Comment:
This will have to be cleaned up in a future commit/sprint. Right now I don't know exactly when to show the new menu, or whether we can delete the old menu.

Reply:
To see the menu, add a session storage variable with this name and give it a non-null value. (`false` doesn't turn it off because it's a string, and therefore not falsy.) You can add this variable in Chrome dev tools, Application tab.